snappy/handlers.go

```diff
@@ -147,3 +168,34 @@ func (h *Handler) MakeMuxer(prefix string) http.Handler {
 
 	return m
 }
+
+// TODO: refactor this copy from cmd/snapweb
```

Comment:
Pardon my ignorance but would it make sense to import this from cmd/snapweb ? I.e. could we do that now or are there some more complications?